### DISCUSSION TITLE
有大佬说下怎么用node命令在windows运行吗?

### DISCUSSION THREAD

**Discussion by YiYuan196** [1715173329/UnblockNeteaseMusic#1223] [Q&A]
### Bug 描述

这个怎么用node在windows上运行呢 像nondanee的可以用命令 node app.js -a 127.0.0.1 -p 80 -f IP
但是这个用node命令好像不行

### 启动命令及环境变量

求大佬教下

### 日志内容

求大佬教下

### 网易云音乐版本号

2.9.6

### 操作系统

windows

### 问题排查

- [X] 我确认我使用的核心是由 UnblockNeteaseMusic 项目官方发行，不是其他任何 fork。
- [X] 我确认我已经升级到了最新的核心版本（推荐使用最新构建而不是 release）。
- [X] 我确认我已经启用了 HTTPS 端口。
- [X] 我确认我已经正确设置了 EndPoint。
- [X] 我确认我已经在对应的客户端正确安装了 CA 证书。

**Comment by mario6714** [comment]
You can use this  "unblockneteasemusic-win-x64.exe " from the releases.

open a terminal (cmd) and for example use the next commands:

I use a bat file for that:

cd C:\Users\ "user" \Downloads

set LOG_LEVEL=debug
set ENABLE_FLAC=true
set ENABLE_LOCAL_VIP=true


unblockneteasemusic-win-x64 -o kuwo -e https://music.163.com -f 103.126.92.132 

[Unblockneatse_example.zip](https://github.com/UnblockNeteaseMusic/server/files/12480040/Unblockneatse_example.zip)

**Comment by ChthoIly** [comment]
1.确保你安装了Node JS。
2.下载项目源码并**解压**。
3.使用**Windows Command Prompt**导航到项目文件夹目录下。
4.在**cmd**中输入`npm install`等待其完成部署。
5.此时你可以在**cmd**中执行`node app.js`来运行脚本或者`node nw.js`来注册为Windows Service。

如果你需要指定对应的端口或者设置其他参数你可以以文本格式直接编辑项目目录下的app.js或者nw.js文件来设置对应的参数。
